exporters/sender/jdk/src/test/java/io/opentelemetry/exporter/sender/jdk/internal/JdkHttpSenderTest.java

```diff
@@ -99,7 +99,10 @@ void testShutdownException() throws Exception {
   @Test
   void sendInternal_RetryableConnectTimeoutException() throws IOException, InterruptedException {
     assertThatThrownBy(() -> sender.sendInternal(new NoOpMarshaler()))
-        .isInstanceOf(HttpConnectTimeoutException.class);
+        .satisfies(
+            e ->
+                assertThat((e instanceof HttpConnectTimeoutException) || (e instanceof IOException))
+                    .isTrue());
```

Comment:
What about the changes to the source code require a change to this or `RetryInterceptorTest`?